project1/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Cormorant+Garamond:400,600|Lexend+Mega&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+
+
+
+
+    <div class="wrapper">
+    <h1>The Crane Wife</h1>
+    <h2><a href="https://www.theparisreview.org/blog/author/cjhauser/">By CJ Hauser</a> &ensp;  July 16, 2019</h2>
+
+    <blockquote>“I was going to do the squirrel but then I thought, that just isn’t CJ. <i>This</i> is CJ.”</blockquote>
+
+<p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the whooping crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for nylon hiking pants that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly at home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be doing that requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and floppy hat featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that looks like this when something in my life has gone so terribly wrong.</p>
+
+<p>Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate New York house with two willow trees I had bought with my fiancé.</p>
+
+<p>Ten days later and I didn’t want to do anything I was supposed to do.</p>
+
+<center>*</center>
```

Comment:
The `center` element is an old solution and is considered obsolete: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

Instead, consider putting the asterisk inside a `div` and, in your CSS, giving that element `text-align: center;`